ChangeLog.d/remove-rsa-mode-parameter.txt

```diff
@@ -0,0 +1,6 @@
+API changes
+   * Remove mode parameter from RSA functions. All encryption,
+     decryption, sign and verify functions are affected. Also
+     removes the RNG parameters from the RSA verify functions.
+     Existing user code which utilises these RSA functions must
+     remove the mode parameter. Fixes #4278.
```

Comment:
“Fixes” is useful for a bug, but not meaningful for an internally-decided enhancement.